02-peer-protocol.md

```diff
@@ -630,6 +634,12 @@ reply with the same value (completing after three messages).
     2. data:
         * [`u64`:`min_fee_satoshis`]
         * [`u64`:`max_fee_satoshis`]
+    1. type: 2 (`local_musig2_pubnonce`)
```

Comment:
In the "extension" style, I think we want to instead add these new TLV fields and requirements to the _new_ document.

Reply:
I am of two minds here. On one hand it's nice to have things in a new document, but I also think it would be confusing to split up `closing_signed` requirements between multiple files without any sort of reference (hyperlink?) from one file to the other